### PR TITLE
it is roman project

### PR DESCRIPTION
**Pull request by fatiabdulahi**
It is the Roman project I work with index.html -styles.css and script.js

### CODE REVIEW

**Comment by fatiabdulahi** [comment]
@sharifarezaie .hello dear sharifa  I compleated my project please review and shear your opinian .thank you

**Comment by Sharifarezaie** [comment]
Waw  l think  tath  is  great 👍 👌 l  approved your project 🥳🥳💯

**Comment by fatiabdulahi** [comment]
@ahzamir Hello dear Sir I hope you are doing well I have copleated my project and my partner is aproved .please review  and shear your opinian .thank you

**Comment by fatiabdulahi** [comment]
@sharifarezaie .thank you dear sharifa

File: README.md [hunk 1]
```diff
@@ -1 +1,288 @@
...
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  updateUI();
+});
+
+convertButton.addEventListener('click', () => {
+  updateUI();
+});
+```
```
Comment (ahzamir) [inline README.md]: - Remove all of this as it is so long.

**Review by ahzamir — APPROVED** [review]
### Hi @fatiabdulahi 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- [OPTIONAL] - In the mobile version your design is great but has a little bit of horizontal scroll, your page should not scroll horizontally so solve this when you have time.
- Check the below comments for your README file solve them and then directly merge your pull request.
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md (hunk 1 above)
Comment (fatiabdulahi) [reply README.md]: @ahzamir thank you dear Sir for your amazing comment .I have remove my code from readme.

**Pull request merged**